lib/typescript/types/content.ts

```diff
@@ -32,8 +32,13 @@ export interface Text {
   text: string;
 }
 
+export interface SuggestionResponse {
```

Comment:
Sorry: I forgot to delete this file. It only really makes sense to be on `types` if it's a universal content model, but since we don't have that anymore I would add this type directly to the code where it's needed